src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -499,7 +499,13 @@ public function getEmail()
                 //ffs, some people don't have an eduroam either.
                 $eduroam = $this->getEduroam();
                 if (empty($eduroam)) {
-                    return;
+                    if (!empty($this->email)) {
+                        //This is an email not associated to a user.
+                        return $this->email;
+                    } else {
+                        //Give up trying to send.
+                        return;
```

Comment:
This seems like a footgun to me - are we sure that all code that calls `$user->getEmail()` can handle the empty case? AFAICT no other branch returns nothing so I'm slightly concerned here

Reply:
Yup, this is how I found this bug in the first place. It tries to send an email with <> as the email address, so just fails and sticks in the MyRadio email queue.

Reply:
Are there any cases in which the new code could cause that failure mode as well?